### ISSUE TITLE
Unit tests need to be revised, and created for `model_training.py` and `get_unique_craters.py`

### ISSUE THREAD

**Issue by cczhu** [silburt/DeepMoon#6]

**Comment by silburt** [comment]
Wrote unit test for get_unique craters.py to extract craters from modelpreds. I was going to write another unit test that converts pixel coordinates to degree/km coordinates, but @cczhu it looks like you've already done this?

**Comment by cczhu** [comment]
Agreed that having two files that do almost the same thing is pretty clunky.  I tried to streamline it into one file, and created some tests to go along with it.  Please see pull request #11.

I did check that `model.summary()` looks the same in either Keras 1 or 2 using your code and my pull request, but didn't check if the entire script runs properly or not.  Would you be able to check that?  Also, it would be great if we could check something basic, like if you throw in a fake training set of all 1s or something, then set the Keras seed to a fixed value, that you get exactly the same output in your code and mind.  That kind of test could then go in the new `test_model_train.py` file.

**Comment by cczhu** [comment]
I did some further refinements to `extract_unique_craters`, and shifted some of your tests into `test_utils.py` in PR #10.
